DataFormats/MuonReco/interface/EmulatedME0Segment.h

```diff
@@ -0,0 +1,77 @@
+#ifndef ME0RecHit_EmulatedME0Segment_h
+#define ME0RecHit_EmulatedME0Segment_h
+
+/** \class EmulatedME0Segment
+ *  Describes a simulated track segment in a z-plane modeling an ME0 chamber.
+ *  It is modeled after CSCSegment, so it is a 4-dimensional object ( origin (x,y) , direction (x,y) )
+ *  Formally it must be defined as a LocalPoint and LocalError but we actually use the global coordinate system.
+ *  
+ *  $Date: 2013/04/22 22:41:32 $
+ *  
+ *  \author David Nash
+ */
+
+#include <DataFormats/TrackingRecHit/interface/RecSegment.h>
+
+#include <iosfwd>
+
+class EmulatedME0Segment : public RecSegment {
+
+public:
+
+    /// Default constructor
+ EmulatedME0Segment() : theOrigin(0,0,0), theLocalDirection(0,0,0), theCovMatrix(4,0),theChi2(0.) {}
+	
+    /// Constructor
+    EmulatedME0Segment(LocalPoint origin, LocalVector direction, AlgebraicSymMatrix errors, double chi2);
+  
+    /// Destructor
+    virtual ~EmulatedME0Segment();
+
+    //--- Base class interface
+    EmulatedME0Segment* clone() const { return new EmulatedME0Segment(*this); }
+
+    LocalPoint localPosition() const { return theOrigin; }
```

Comment:
all comments about addition of virtual made in ME0Segment.h
 apply here as well